include/zenoh-pico/api/handlers.h

```diff
@@ -51,9 +51,15 @@
             return;                                                                                                  \
         }                                                                                                            \
         if (elem == NULL) {                                                                                          \
-            internal_elem->_rc.in = NULL;                                                                            \
```

Comment:
can we just use standard _clone methods that we define almost for every owned type, to avoid duplicating handlers code for rc and not-rc ?